specification/authorization/resource-manager/Microsoft.Authorization/preview/2020-10-01-preview/examples/GetRoleAssignmentScheduleInstancesByScope.json

```diff
@@ -7,6 +7,7 @@
   "responses": {
     "200": {
       "body": {
+        "nextLink": "https://getMoreData.net/foo",
```

Comment:
We have added next link in the model but that's only for future compatibility - we do not send it in the response currently that's why I did not have it in the examples. I believe you can remove this and the validations will still pass as they were passing earlier.

Reply:
removed